src/test/java/core/basesyntax/service/impl/StorageServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.StorageService;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageServiceImplTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private static final String NON_EXISTENT_FRUIT = "orange";
+    private static final int BANANA_QUANTITY = 50;
+    private static final int APPLE_QUANTITY = 35;
+    private static final int NEGATIVE_QUANTITY = -15;
+    private static final String QUANTITY_ERROR_MESSAGE =
+            "Quantity must be a non-negative value, but you provided: ";
+    private StorageService storageService;
+
+    @BeforeEach
+    void setUp() {
+        storageService = new StorageServiceImpl();
+    }
+
+    @Test
+    void add_validDataToStorage_Ok() {
+        storageService.add(BANANA, BANANA_QUANTITY);
+        assertEquals(BANANA_QUANTITY, storageService.get(BANANA));
+    }
+
+    @Test
+    void add_NegativeQuantityToStorage_NotOk() {
+        String expectedMessage = QUANTITY_ERROR_MESSAGE + NEGATIVE_QUANTITY;
+        var exception = assertThrows(IllegalArgumentException.class, () ->
+                storageService.add(BANANA, NEGATIVE_QUANTITY));
+        assertEquals(expectedMessage, exception.getMessage());
```

Comment:
Please introduce some spaces to make code more readable, I suggest to separate following parts: GIVEN, WHEN, THEN